sample-linux.ini

```diff
@@ -10,6 +10,10 @@ maxCacheEntries = 2000000
 mruSymbolStateFile = /tmp/snappy-mru-symbols.json
 maxMRUSymbolsPersist = 10
 
+[DiskCache]
+cachePath = /tmp/snappy/cache
+maxCacheFiles = 50
```

Comment:
50 is way too low, even a single request can have 50 libs. 

1. How big is one parsed+pickled xul.sym file?
2. Determine the maxCacheFiles limit based using max ~500MB of storage space filled with parsed+pickled xul.sym files.
3. How would the server have behaved in the situation when there were 60 libs needing to be stored (for one request) but the quota was 50?